Sports.py

```diff
@@ -2,49 +2,248 @@
 #
 # Title:      Sports Display 64x128 LED Matrix
 # Author:     Timothy Kosinski
-# Date:       06MAR2024
+# Date:       12APR2024
 # Description:
 #  Drives my LED Matrix Panel, Adifruit Display
+# Takes In NBA Player Names and Displays Players Stats
+# By using api_calls.py (API) to run the display
 #==========================================================
 
 #!/usr/bin/env python
 from samplebase import SampleBase
 from rgbmatrix import graphics
 from datetime import datetime
+from PIL import Image, ImageOps, ImageEnhance 
+
+from api_calls import GameFacade
+import api_calls
+import http.client
 import time
+import socket
+import os
+
+class UserInfo:
+    def __init__(self,ip_address):
+        self.ip_address = ip_address
+        
+    def update_userInfo(self,ip_address):
+        self.ip_address = ip_address
 
 class PlayerStats:
-    def __init__(self, name, team, jnumber):
+    def __init__(self, name, team ,points ,assists, rebounds, blocks, steals, field_goals_percent, three_pointers_percent, free_throws_percent):
+        self.name = name
+        self.team = team
+        self.assists = assists
+        self.points = points
+        self.rebounds = rebounds
+        self.blocks = blocks
+        self.steals = steals
+        self.field_goals_percent = field_goals_percent
+        self.three_pointers_percent = three_pointers_percent
+        self.free_throws_percent = free_throws_percent
+    
+
+    def update_stats(self,name,team,points,assists, rebounds, blocks, steals, field_goals_percent, three_pointers_percent, free_throws_percent):
         self.name = name
         self.team = team
-        self.jnumber = jnumber
+        self.points = points
+        self.assists = assists
+        self.rebounds = rebounds
+        self.blocks = blocks
+        self.steals = steals
+        self.field_goals_percent = field_goals_percent
+        self.three_pointers_percent = three_pointers_percent
+        self.free_throws_percent = free_throws_percent
+## End of player class ## 
+
+class TeamStats:
+    def __init__(self, team ,points ,assists, rebounds, blocks, steals, field_goals_percent, three_pointers_percent, free_throws_percent):
+        self.team = team
+        self.assists = assists
+        self.points = points
+        self.rebounds = rebounds
+        self.blocks = blocks
+        self.steals = steals
+        self.field_goals_percent = field_goals_percent
+        self.three_pointers_percent = three_pointers_percent
+        self.free_throws_percent = free_throws_percent

```

Comment:
is this implemented? do we use team stats?

Reply:
I was was wondering I believe since importing from api calls he could append to team stats there, not 100%.